dockerfiles/startall.sh

```diff
@@ -21,6 +21,7 @@
 # Usage startup.sh [<image.version>] [<helm-install-options>]
 
 DOCKER_REPO=hpccsystems
+scriptdir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
```

Comment:
Looks like a good change, but not really related to the rest of the PR...

Reply:
It is because my test scripts are in a different directory, and they were calling this one.  Do you want it in a separate PR?

Reply:
You can leave it in here